src/c++/perf_analyzer/load_worker.h

```diff
@@ -56,8 +56,9 @@ class LoadWorker : public IWorker {
       std::uniform_int_distribution<uint64_t>& distribution,
       std::condition_variable& wake_signal, std::mutex& wake_mutex,
       bool& execute)
-      : thread_stat_(thread_stat), parser_(parser), data_loader_(data_loader),
-        factory_(factory), sequence_stat_(sequence_stat),
+      : id_(id), thread_stat_(thread_stat), parser_(parser),
```

Comment:
It looks like id_ is not currently used--should this addition be held off?

Reply:
removed -- although it is very helpful for debug. Hopefully soon it will be pulled out of the thread_config and into this base class.

Reply:
actually, the plumbing was already there from the parents. It is more work for me to rip this out. It is helpful for debug and will help us to decouple the thread_config from the child classes

Reply:
Also, I think it would be needed for your sequence class? For a thread to be able to identify themselves?